lib/internal/modules/esm/resolve.js

```diff
@@ -461,7 +461,7 @@ function resolvePackageTargetString(
       } catch {}
       if (!isURL) {
         const exportTarget = pattern ?
-          StringPrototypeReplace(target, patternRegEx, subpath) :
+          StringPrototypeReplace(target, patternRegEx, () => subpath) :
```

Comment:
```suggestion
          RegExpPrototypeReplace(patternRegEx, target, () => subpath) :
```

Reply:
I'm not sure there is a RegExPrototypeReplace?

Reply:
Sorry, my mistake. (the reason for using it can be found at https://github.com/nodejs/node/pull/38635/files#diff-1fd7756e2222fdc2c8dbd60f718237a78233a8697d418ec9044e167b673bcf96R519-R528)
```suggestion
          RegExpPrototypeSymbolReplace(target, patternRegEx, () => subpath) :
```

Reply:
Ah I see. I don't actually see any justification there though other than it just being another documented form?

Reply:
Ah ok got it.